File: 1050/1050.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var ddd_array = {
  61: "Brasilia",
  71: "Salvador",
  11: "Sao Paulo",
  21: "Rio de Janeiro",
  32: "Juiz de Fora",
  19: "Campinas",
  27: "Vitoria",
  31: "Belo Horizonte",
};

var a = parseInt(lines.shift());

if (ddd_array[a]) {
  console.log(ddd_array[a]);
} else {
  console.log("DDD nao cadastrado");
}

File: 1050/1050.lua
local ddd_table = {
    [61] = "Brasilia",
    [71] = "Salvador",
    [11] = "Sao Paulo",
    [21] = "Rio de Janeiro",
    [32] = "Juiz de Fora",
    [19] = "Campinas",
    [27] = "Vitoria",
    [31] = "Belo Horizonte",
    
}


local a = io.read("*n")

if ddd_table[a] ~= nil then
    print(ddd_table[a])
else
    print("DDD nao cadastrado")
end